funcs/cloudrun/cloudrun.go

```diff
@@ -0,0 +1,89 @@
+package cloudrun
```

Comment:
As I explained in the call, it would be great to also define this interface for CF. The actual Go interfaces will be the same, but `Run*` will be different.

Reply:
Discussed.